docs/src/examples/components/Accordion/BestPractices/AccordionBestPractices.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react'
+
+import ComponentBestPractices from 'docs/src/components/ComponentBestPractices'
+
+const doList = [
+  {
+    key: 'one',
```

Comment:
may suggest to omit keys from the client's API - we could safely use items' indexes